.github/workflows/ci_dockers.yml

```diff
@@ -95,24 +95,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # the config used in '.github/workflows/ci_test-conda.yml'
-        python_version: ["3.8"]
-        pytorch_version: ["1.8", "1.9", "1.10"]
+        include:
+          # see: https://pytorch.org/get-started/previous-versions/
+          - {python_version: "3.8", pytorch_version: "1.8", cuda_version: "11.1"}
+          - {python_version: "3.8", pytorch_version: "1.9", cuda_version: "11.1"}
+          - {python_version: "3.8", pytorch_version: "1.10", cuda_version: "11.1"}
+          - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
```

Comment:
This PR can be merged but the 1.11 job is failing

Reply:
yes, but that will be fixed by creating this new docker image...